### DISCUSSION TITLE
Reduce class channels cooldown on Discord.

### DISCUSSION THREAD

**Discussion by minthes** [chromiecraft/chromiecraft#6074] [Suggestions]
I understand the rationale behind having cooldowns on class channels, as they encourage users to stay on-topic within those channels and to be pooled in #general and #off-topic for everything else. To enhance communication and collaboration within Discord class channels, I suggest considering a reduction in the cooldown period to a reasonable and unobtrusive timeframe, such as 5-15 seconds.

**Comment by heyitsbench** [comment]
Even 30s is substantially better than 2m.

**Comment by Peti321** [comment]
It should really be lower than a minute.

**Comment by 55Honey** [comment]
Hi @minthes 
Thanks for your suggestion.
The original cool down for posts in the theory crafting channels was a full hour.
It's already been reduced to 15m and 2m respectively. These reductions have caused a severe negative impact on the average quality of the posts.

There are no plans to reduce it further as of now.

**Comment by ghost** [comment]
I guess I can just edit 2 cat gifs into each post.

**Reply by Never6** [reply]
yes more cat gifs :)
